src/util/parse.js

```diff
@@ -23,15 +19,54 @@ export const parse = async (options) => {
     options.flavour = options.flavor;
   }
```

Comment:
I didn't notice lines 18-20 before. But they are now redundant due to line 25 and can be removed.